src/hooks/useHover/__stories__/useHover.stories.mdx

```diff
@@ -0,0 +1,58 @@
+import { Canvas, Meta, Story } from "@storybook/addon-docs";
+import useHover from "../useHover";
+import UsageGuidelines from "../../../storybook/components/usage-guidelines/usage-guidelines";
+import "../../__stories__/general-hooks-stories.scss";
+
+<Meta title="Hooks/useHover" />
+
+# useHover
+
+- [Overview](#overview)
+- [Arguments](#arguments)
+- [Returns](#returns)
+- [Feedback](#feedback)
+
+## Overview
+
+Attaches a listener for <code>mouseover</code> and <code>mouseout</code> DOM events on a specific element, returning <code>isHovered</code> state.
```

Comment:
I think that this explanation is too technical and about the implementation details 
What about something like: 
```suggestion
Detect whether the mouse is hovering an element by returning is <code>isHovered</code> state.
```
feel free to change it according to what you think is best.